main.go

```diff
@@ -24,8 +24,10 @@ var (
 	threshold  = flag.Int("threshold", DefaultThreshold, "minimum token sequence as a clone")
 	serverPort = flag.String("serve", "", "run server at port")
 	addrs      AddrList
-	html       = flag.Bool("html", false, "html output")
 	files      = flag.Bool("files", false, "files names from stdin")
+
+	html     = flag.Bool("html", false, "html output")
+	plumbing = flag.Bool("plumbing", false, "plumbing output")
```

Comment:
I would suggest elaborating on the meaning here, e.g., "plumbing output for consumption by scripts or tools" or so.